flake.nix

```diff
@@ -35,7 +42,10 @@
           };
         });
   in {
-    nixosModules.default.imports = [./nixarr vpnconfinement.nixosModules.default];
+    nixosModules.default.imports = [
+      ((import ./nixarr) inputs)
```

Comment:
Is there a way to use `specialArgs` to pass around `unmanic-nix` as an extra module arg alongside `lib, config, pkgs`? I know for flake `.nixosConfigurations` you can do something like this:
```nix
outputs = {
  self,
  NixVirt,
  nixpkgs,
  ...
}: {
  nixosConfigurations.blah = nixpkgs.lib.nixosSystem {
    modules = [ blah.nix ];
    specialArgs = {
      inherit NixVirt;
    };
  };
};
```
And then `NixVirt` is available within `blah.nix` alongside the usual stuff:
```nix
{
  config,
  lib,
  pkgs,
  NixVirt,
  ...
}: { ... }
```